Formula/alure.rb

```diff
@@ -3,7 +3,7 @@ class Alure < Formula
   homepage "https://kcat.strangesoft.net/alure.html"
   url "https://kcat.strangesoft.net/alure-releases/alure-1.2.tar.bz2"
   sha256 "465e6adae68927be3a023903764662d64404e40c4c152d160e3a8838b1d70f71"
-  revision 1
+  revision 2
```

Comment:
Since you're only changing the `test` block, a revision bump should not be needed.

Reply:
Thanks, fixed.